docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+.NET Standard Library
+=====================
+
+The .NET Standard Library is a formal specification of .NET APIs that are intended to be available on all .NET runtimes. The motivation behind the Standard Library is establishing greater uniformity in the .NET ecosystem. [ECMA 335](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) continues to establish uniformity for .NET runtime behavior, but there is no similar spec for the .NET Base Class Libraries (BCL) for .NET library implementations. 
+
+The .NET Standard Library is a standardized BCL, enabling two key scenarios: 
+
+- Provide a baseline set of BCL APIs on all .NET platforms, independent of workload.
+- Enable developers to produce portable libraries that are usable across .NET runtimes, using this same set of APIs.
+
+The second scenario is the same as PCL. .NET Standard Library can be thought of as the next generation of [Portable Class Libraries (PCL)](https://msdn.microsoft.com/library/gg597391.aspx). The .NET Standard Library improves on the experience of creating portable libraries by curating a standard BCL and establishing greater uniformity across .NET runtimes as a result. 
+
+The spec is called "The .NET Standard Library". The more general ecosystem that builds on top of the spec is referred to as ".NET Standard" or coloquially as "NETStandard". A Library that targets the spec is called "a .NET Standard library (NSL)" or coloquially as "a NETStandard library" or as "an NSL" (analogue of "PCL").
+
+Specification
+=============
+
+The .NET Standard Library spec is a standardized set of APIs. A given .NET runtime will implement and advertise support for a particular version of the .NET Standard Library. That runtime is then able to support source and binaries that target that and previous versions of the spec.
+
+The spec is maintained by .NET runtime implementors, specifically Microsoft (includes .NET Framework, .NET Core and Mono) and Unity. A public feedback process is used as part of establishing new .NET Standard Library versions.
+
+Official Artifacts
+------------------
+
+The official spec is a set of .cs files that define the APIs that are part of the standard. The [ref directory](https://github.com/dotnet/corefx/tree/master/src/System.Runtime/ref) for each [component](https://github.com/dotnet/corefx/tree/master/src) defines the .NET Standard Library APIs. While the ref artifacts reside in the [CoreFX repo](https://github.com/dotnet/corefx), they are not .NET Core specific.
+
+The [NETStandard.Library](https://github.com/dotnet/corefx/blob/master/pkg/NETStandard.Library/NETStandard.Library.packages.targets) metapackage describes the set of libraries that define (in part) one or more .NET Standard Library versions.
```

Comment:
Ahhh, that makes sense.  Perhaps it's best to include an example, then.  Your System.Runtime example makes it clear.